README.md

```diff
@@ -18,7 +18,7 @@ All spring way edge modules, that will use this library, should add it as a depe
 After that, all requests that will call the edge module will go through the request filter from the library. 
 As a result, incoming request will contain ***x-okapi-token*** in the header, and the edge module will be able to invoke folio modules.
 
-By default, filter will make authorization and receive the ***x-okapi-token*** header only if the requested URL does not start with: /admin/health, admin/info, /_/tenant.
+By default, filter will make authorization and receive the ***x-okapi-token*** header only if the requested URL does not start with: /admin/health, admin/info, /swagger-resources, /v2/api-docs, /swagger-ui, /_/tenant.
```

Comment:
I'm confused by this section of the README.  Edge APIs should require an api key to be provided by a configurable set of sources, e.g. query argument, path parameter, or header.  If you have an X-Okapi-Token there is no need to use an edge API.

Reply:
this part of readme means that filter will work and get okapi-token for all request except those that listed in excludeList

Reply:
> incoming request will contain ***x-okapi-token***
The incoming request should provide an API key, not an X-Okapi-Token.

Reply:
updated